datadotworld/client/_swagger/apis/queries_api.py

```diff
@@ -632,7 +632,7 @@ def execute_query_with_http_info(self, id, **kwargs):
                                         body=body_params,
                                         post_params=form_params,
                                         files=local_var_files,
-                                        response_type=None,
+                                        response_type='PaginatedQueryResults',
```

Comment:
I have no context to review this 😅 but, I think the api response type depends on the request in this case, no?

Reply:
Shoot, didn't link the ticket
https://dataworld.atlassian.net/browse/IN-2775